hack/scale_out_poc/nginx.conf

```diff
@@ -0,0 +1,102 @@
+user www-data;
```

Comment:
missing header

Reply:
This is the template file for nginx.conf file. It should not have the copyright header, just like other templates files, such as etcd.manfiest, etc.